src/lectern-client/schema-functions.ts

```diff
@@ -125,15 +125,17 @@ const convertFromRawStrings = (
       return undefined;
     }
 
-    if (isNotEmptyString(record[field.name])) {
-      // convert field to match corresponding enum from codelist, if possible
-      if (field.restrictions && field.restrictions.codeList) {
-        const formattedField = field.restrictions.codeList.find(
-          e => e.toString().toLowerCase() === record[field.name].toString().toLowerCase(),
-        );
-        if (formattedField) {
-          mutableRecord[field.name] = formattedField;
-        }
+    if (isEmptyString(record[field.name])) {
+      return undefined;
+    }
+
+    // convert field to match corresponding enum from codelist, if possible
+    if (field.restrictions && field.restrictions.codeList) {
+      const formattedField = field.restrictions.codeList.find(
+        e => e.toString().toLowerCase() === record[field.name].toString().toLowerCase(),
+      );
+      if (formattedField) {
+        mutableRecord[field.name] = formattedField;
       }
       return undefined;
```

Comment:
are `codeLists` always strings...

Reply:
they could be numbers as far as I remember in Lectern, I don't think we have an integer code list yet though, plus this is not a code I added now, this is the existing code to normalize the enum format.

Reply:
I was only asking because we return after replacing with enum format, so for enum numbers it will remain string. Yeah but you are right we don't have those so I guess it doesn't matter

Reply:
I was only asking about this since we return undefined after enum replace, so integer/number enum would remain string. But yea your right I gues it doesn't really matter.